phpstan.neon.dist

```diff
@@ -0,0 +1,7 @@
+includes:
```

Comment:
Why the `.dist` extension?

Reply:
because phpcs is configured the same way already

Reply:
Usually you'll have `*.dist` when it makes sense to have local configuration values. For example phpunit might have local env vars for tests to run correctly. I don't think that's the case for phpstan?